.github/release.yml

```diff
@@ -0,0 +1,6 @@
+changelog:
+  exclude:
+    authors:
+      - dependabot # Automated dependency updates
```

Comment:
```suggestion
      - dependabot # Automated dependency updates. Since dependabot is only used for managing github action updates, not the Slicer source code, these changes are excluded from the changelog.
```